tasks/controllers/user.js

```diff
@@ -1,11 +1,16 @@
-import { getFilteredUsers, isUserExist, storage, User } from '../models/user.js';
+import { getFilteredUsers, isUserExist, getUserByLogin, storage, User } from '../models/user.js';
 import { getLimit } from '../utils/util.js';
+import { Authentication } from '../utils/auth.js';
+
+const getErrorNumber = 777;
 
 export const Controller = {
     user: {
         get(request, response) {
             if (isUserExist(request.params.id)) {
                 response.status(200).send(storage.get(request.params.id));
+            } else if (request.params.id > getErrorNumber) {
+                throw Error('unexpected error');
```

Comment:
Consider always storing such strings at a more global level and using the variable reference throughout the codebase.